setup.py

```diff
@@ -33,7 +33,7 @@ def package_data(pkg, roots):
         'sql_grader',
     ],
     install_requires=[
-        'Django',
+        'Django<3.0',
```

Comment:
Was this needed for tests to pass?  We'll need to remove this fairly soon to start testing against 3.0.